web/plugins/meetup-api/meetup-api.php

```diff
@@ -28,11 +28,30 @@
 );
 $provider = new CachedProvider($provider);
 
+// Display `the_content` only if there are no valid events.
+add_filter('wpberlin/website/front_page_no_events', function ($title, $content) use ($provider) {
```

Comment:
I think this filter (as well as the one below), should be put into the theme instead of the plugin. Instead of coupling the theme to the plugin, we should add the plugin as a dependency to the theme (cf. #41)